### PR TITLE
Update tmr-meeting-recording-change.md

### PR DESCRIPTION
**Pull request by pupara**
Changed Skype for Business Connector to Microsoft Teams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/tmr-meeting-recording-change.md [hunk 1]
```diff
@@ -62,20 +62,20 @@ Watch "Meeting Recording" for more information.
...
 
-    b. Follow the prompts to install it.
-
-    c. Restart your machine.
+  a. Install [Teams PowerShell](https://docs.microsoft.com/microsoftteams/teams-powershell-install).
 
 2. Launch PowerShell as an admin
 
-3. Import the SkypeOnline Connector and log in as a Teams admin.
+3. Import the MicrosoftTeams and log in as a Teams admin.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:75]:
```suggestion
3. Import Microsoft Teams and log in as a Teams admin.
```

File: Teams/tmr-meeting-recording-change.md [hunk 2]
```diff
@@ -62,20 +62,20 @@ Watch "Meeting Recording" for more information.
 
 ## Set up the meeting recording option for OneDrive for Business and SharePoint
 
-1. Install the Skype For Business Online PowerShell admin console.
+1. Install the Teams PowerShell module.
 
-    a. Download [Skype for Business Online PowerShell](https://docs.microsoft.com/microsoft-365/enterprise/manage-skype-for-business-online-with-microsoft-365-powershell?view=o365-worldwide).
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:68]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

File: Teams/tmr-meeting-recording-change.md [hunk 3]
```diff
@@ -62,20 +62,20 @@ Watch "Meeting Recording" for more information.
...
+> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector
```
Comment (JohanFreelancer9) [inline Teams/tmr-meeting-recording-change.md:69]:
```suggestion
> If you're using the latest [Teams PowerShell public release](https://www.powershellgallery.com/packages/MicrosoftTeams/), you don't need to install the Skype for Business Online Connector.
```

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cichur** [comment]
Hi @JohanFreelancer9 

 I've shared this with our SME and am awaiting his response. We are reviewing your changes. Thank you for your feedback.

**Comment by cichur** [comment]
Hi @JohanFreelancer9 
I've shared your edits with the SMEs. They are discussing your suggested changes. In the meantime, I've published a few modifications to the article based on your suggested changes, such as adding the note.  - Cindy

**Comment by JohanFreelancer9** [comment]
@cichur - Thank you Cindy, sounds good.

**Pull request closed without merging**